hub/api/tests/test_api_with_compression.py

```diff
@@ -144,3 +144,46 @@ def test_unsupported_compression(memory_ds: Dataset):
 @pytest.mark.xfail(raises=TensorMetaMissingRequiredValue, strict=True)
 def test_missing_sample_compression_for_image(memory_ds: Dataset):
     memory_ds.create_tensor("tensor", htype="image")
```

Comment:
can you add another xfail test below this one for when `sample_compression` and `chunk_compression` are both provided?